### PR TITLE
Add UFOGenScheduler to Community Examples

### PR DESCRIPTION
**Pull request by dg845**
# What does this PR do?

This PR adds a scheduler for the [UFOGen model](https://arxiv.org/pdf/2311.09257.pdf) to the community examples. A separate UFOGen pipeline is not currently implemented because UFOGen is intended to be initialized from a pretrained diffusion model (Stable Diffusion 1.5 is used in the paper) and so `UFOGenScheduler` is designed to be compatible with existing pipelines such as `StableDiffusionPipeline`.

Based on #6133. The code in `scheduling_ufogen.py` is the same as in that PR, except with the `diffusers` imports changed from relative imports to absolute imports.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@patrickvonplaten
@patil-suraj 
@JunbongJang

### CODE REVIEW

**Review by patrickvonplaten — COMMENTED** [review]
Could we also add a short entry to https://github.com/huggingface/diffusers/blob/main/examples/community/README.md

**Comment by dg845** [comment]
Added an entry into the community README for `scheduling_ufogen.py`/`UFOGenScheduler` with a usage example (although the usage example doesn't quite work in the sense that the `ufogen_model_id_or_path` is just a dummy value rather than pointing to a physical trained UFOGen checkpoint, since I'm not aware of any publicly available such checkpoints).
